src/frontend/src/flows/login.ts

```diff
@@ -61,12 +61,12 @@ export const login = async (
         return login(userIntent);
       }
     }
-  } catch (err) {
+  } catch (err:any) {
     await displayError({
       title: "Something went wrong",
       message:
-        "An unexpected error occurred during authentication. Please try again",
-      detail: err,
+        "An unexpected error occured during authentication. Please try again",
```

Comment:
Mh, I don't think that's correct: 

> Spelling words isn’t always easy, especially when they contain double letters. In this case, the correct spelling is occurred. How can you remember it when you need it?
> 
> The best way to remember how to spell occurred is to remember its double set of double consonants. In English, the final letter is doubled when a word of two or more syllables has stress on the final syllable. Occur fits the rule, so there are two Cs and two Rs in occurred.

https://www.grammarly.com/blog/occurred-occured-ocurred/

Reply:
Woops, I think that happened from a merge conflict. Updated